.github/workflows/chatgpt-review.yml

```diff
@@ -0,0 +1,18 @@
+name: ChatGPT Review
+
+on:
+  pull_request_target:
+    types: [opened, reopened, edited, ready_for_review]
+    branches:
+      - 'main'
+
+jobs:
+  chatgpt-review:
+    name: ChatGPT Review
+    runs-on: ubuntu-latest
+    steps:
+    - uses: feiskyer/ChatGPT-Reviewer@v0.3
+      name: ChatGPT Review
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@Tatsinnit Thanks for this awesome PR. I like the idea like we discussed the other day. Only catch is we need to add these tokens to the repository for this GH action to work.